utils/build-script

```diff
@@ -856,6 +724,174 @@ class BuildScriptInvocation(object):
 
         return options
 
+
+class BuildScriptInvocation(object):
+
+    """Represent a single build script invocation."""
+
+    @staticmethod
+    def validate_arguments(toolchain, args):
+        if toolchain.cc is None or toolchain.cxx is None:
+            diagnostics.fatal(
+                "can't find clang (please install clang-3.5 or a "
```

Comment:
I don't think that we should hardcode the version here, its too easy to forget to update it.

Reply:
This code is just a copy of the existing code. You are right that hardcoding '3.5' in the string is not the best idea, but I don’t think there's a good way (right now) to do it better.

The accepted versions are “hardcoded” in `swift_build_support.toolchain`, and they are not easy to query. I can say that 3.5 seems to still be the minimum version (but this is only the executable suffix, so…).